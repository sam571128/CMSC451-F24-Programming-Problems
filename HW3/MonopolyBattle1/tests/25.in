12 22 299
9 1
1 5
1 3
2 4
6 4
2 5
3 6
12 2
9 6
11 1
7 2
1 7
12 8
10 8
10 12
8 12
1 2
6 3
10 10
3 7
4 7
1 1
