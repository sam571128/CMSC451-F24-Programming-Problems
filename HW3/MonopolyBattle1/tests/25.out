22 20 18 18 21 22 21 -1 -1 -1 -1 -1 